### DISCUSSION TITLE
Elastic search warn ingest FortiGate logs

### DISCUSSION THREAD

**Discussion by eversafe-leandro** [Security-Onion-Solutions/securityonion#12536] [Unsupported Versions] [answered]
Need help with that problem:
[2024-03-08T00:13:24,906][WARN ][logstash.outputs.elasticsearch] Could not index event to Elasticsearch. {:status=>400, :action=>["create", {:_id=>nil, :_inde                           x=>"logs-fortinet_fortigate.log-default"

Has this ever happened to any of you? where did you find the solution?

**Comment by eversafe-leandro** [comment]
"error"=>{"type"=>"document_parsing_exception", "reason"=>"[1:5756] failed to parse field [event.module] of type 
[constant_keyword] in document with id 'mtMcH44BFDsl9ReZcjn1'. Preview of field's value: 'fortinet_fortigate'", "caused_by"=>{"type"=>"illegal_argument_exception", "reason"=>"[constant_keyword] field [
event.module] only accepts values that are equal to the value defined in the mappings [fortinet], but got [fortinet_fortigate]"

**Comment by eversafe-leandro** [comment]
I saw in forum that fortigate 7.0 + don't suport..

**Reply by cm-ops** [reply, marked as answer]
Some have done this - https://github.com/Security-Onion-Solutions/securityonion/discussions/11995#discussioncomment-7849553
